enterprise/frontend/src/embedding/data-picker/DataSelector/tests/DataSelector.unit.spec.js

```diff
@@ -9,7 +9,7 @@ import {
   createSampleDatabase,
 } from "metabase-types/api/mocks/presets";
 
-import { UnconnectedDataSelector as DataSelector } from "./DataSelector";
+import { UnconnectedDataSelector as DataSelector } from "../DataSelector";
```

Comment:
so I moved `DataSelector` under its own directory, so I could add `tests` directory which would hold a new TypeScript test file since I don't wanna add new tests in `DataSelector.unit.spec.js` since typing doesn't work there.